sr_robot_launch/scripts/construct_robot_description

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+# Copyright 2019 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import rospy
+import rospkg
+from subprocess import check_output
+from sr_ur_arm_calibration_loader.sr_ur_arm_calibration_loader import SrUrLoadCalibration
+
+
+class SrRobotDescriptionExceptions(Exception):
+    pass
+
+
+class PrefixNotCorrect(SrRobotDescriptionExceptions):
+    pass
+
+
+class SrConstructRobotDescription():
+    def __init__(self):
+        self._robot_description_file = rospy.get_param("~robot_description_file")
+        CONST_ARM_TYPE = rospy.get_param("~arm_type")
+        self._side = ''
+        self._robot_description_params = {}
+        self._CONST_RA_ARM = {'prefix': 'ra', 'ip_address': '10.8.1.1', 'arm_type': CONST_ARM_TYPE}
```

Comment:
I'm hesitant to mention this, but it would be good to fetch these IPs automatically. From what I can tell, these IPs should have been pushed to the parameter server (under `/ra_sr_ur_robot_hw/robot_ip`, [loaded here](https://github.com/shadow-robot/sr_interface/blob/9ceb11dbdbe92a9cf2f19b315e087755f3a1e31c/sr_robot_launch/launch/sr_hardware_control_loop.launch#L166)) by the time this node has started. We could add a 'wait for param' here and fetch the IP(s) from the parameter server? What do you think?

Reply:
We will create a user story for this :)